tests/test_equations.py

```diff
@@ -107,6 +122,49 @@ def test_integral():
     )
 
 
+def test_Integral():
```

Comment:
```suggestion
def test_integral():
```

Reply:
Thanks, I also changed test_Piecewise() to test_piecewise()